packages/ember-metal/lib/meta.js

```diff
@@ -345,6 +346,14 @@ export class Meta {
       obj[key] = value;
     }
   }
+
+  setFactory(factory) {
```

Comment:
`get` / `set` seem fine to me, I just _slightly_ dislike the [babel transpilation](http://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015-loose&targets=&browsers=&builtIns=false&debug=false&code=class%20Foo%20%7B%0A%20%20get%20derp()%20%7B%20%7D%0A%20%20set%20derp(value)%20%7B%20%7D%0A%7D):

```js
// input

class Foo {
  get derp() { }
  set derp(value) { }
}

// output
var Foo = function () {
  function Foo() {
    _classCallCheck(this, Foo);
  }

  _createClass(Foo, [{
    key: "derp",
    get: function get() {},
    set: function set(value) {}
  }]);

  return Foo;
}();
```